### ISSUE TITLE
Bug found in code - parameter name incorrect

### ISSUE THREAD

**Issue by fenilgmehta** [linuxmint/warpinator#153]
Check this line - https://github.com/linuxmint/warpinator/blob/1773127d1f7fd5fca20610ed655df104e25177a4/src/transfers.py#L224
It should be `fatal` instead of `fatel`

**Comment by fenilgmehta** [comment]
I think PyCharm should be used for development as it automatically detects such issues, and provides many more features.

**Comment by mtwebster** [comment]
Thanks.

We use a number of different languages, not just python, so a more generalized editor is preferred (I've used sublime for a long time).

Also, the couple of times I tried something like PyCharm, it wasn't very useful due to how we use python in most of our software - most 'modules' are *not* actually python modules but C libraries with language bindings for python, or some other language - if I recall, they didn't really know what to do with this stuff. Maybe this has changed more recently, but I still don't want to use multiple editors for different languages.
